src/robotFactory.js

```diff
@@ -1,10 +1,88 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords = {
+    x: 0, y: 0,
+  }, chipVersion) {
+    if (!coords.hasOwnProperty('x')) {
+      coords.x = 0;
+    };
 
-class FlyingRobot {}
+    if (!coords.hasOwnProperty('y')) {
+      coords.y = 0;
+    };
 
-class DeliveryDrone {}
+    this.name = name;
+    this.weight = weight;
+    this.coords = coords;
+    this.chipVersion = chipVersion;
+  }
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
+
+  goBack(step = 1) {
+    this.coords.y -= step;
+  }
+
+  goRight(step = 1) {
+    this.coords.x += step;
+  }
+
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  }
+  getInfo() {
```

Comment:
```suggestion
  goLeft(step = 1) {
    this.coords.x -= step;
  }
  
  getInfo() {
```

Add empty line between methods